tests/integration/update.bats

```diff
@@ -48,6 +48,7 @@ function check_cgroup_value() {
 }
 
 @test "update" {
+    requires cgroups_kmem
```

Comment:
Yeah, maybe split into several functions like:
```
update_blkio()
update_cpu()
update_cpuset()
update_memory()
```

But I think we can do it in a separate PR.